models.py

```diff
@@ -1,47 +1,79 @@
-from django.db import models
+import secrets
 
-# Create your models here.
 from django.contrib.auth.models import User
+from django.db import models
+
+CATEGORIES = ((1, 'Trinken'), (2, 'Snacks'), (3, 'Eis'))
+
+
+def generate_key():
+    while True:
+        key = secrets.token_hex(32)
+        if not Purchase.objects.filter(key=key).exists():
+            break
+
+    return key
 
-CATEGORIES = (
-		('D', 'Trinken'),
-		('F', 'Snacks'),
-		('I', 'Eis'),
-	)
```

Comment:
Will changing the short identifiers break anything with existing installations?

Reply:
Described in the new migrations push.